tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
```

Comment:
Let's also add `allowSyntheticDefaultImports`. Can see [what it means here](https://www.typescriptlang.org/docs/handbook/compiler-options.html) but basically makes TypeScript behave more like Babel when doing imports. 

i.e. you can do `import React from 'react'` instead of `import * as React from 'react'`.

Reply:
Also need `"moduleResolution": "node". [More details here](https://www.typescriptlang.org/docs/handbook/module-resolution.html)

Reply:
Got it!